nc/source/ecs/ColliderTree.cpp

```diff
@@ -82,29 +83,28 @@ namespace nc::ecs
         entries.reserve(DensityThreshold);
     }
 
-    void Octant::BroadCheck(const DirectX::BoundingSphere& dynamicEstimate, std::vector<const StaticTreeEntry*>* out) const
+    void Octant::BroadCheck(physics::SphereCollider collider, std::vector<const StaticTreeEntry*>* out) const
```

Comment:
For when this is no longer fresh in my mind, will you rename collider here to broadCollider or estimatedCollider or something of that nature so that I will not look at this later and think "Why are we only doing this for Sphere Collider?"